lang/ftidy/Portfile

```
@@ -25,63 +26,29 @@ long_description \
     of keywords or non-keywords, and do various other clean-up \
     tasks on FORTRAN-77 source programs.
 
-checksums \
-    md5 f940dfa793408368f92af6aab535619b \
-    sha1 200e4531f172355cde49c455e2f7e9df8f1f8404
+checksums md5    a2611f7448119f9f0982ddfd3e68140c \
+          sha1   82da71cac7380016233ab2aa217dbf3a0dd3686f \
+          rmd160 1b96ed6a8e0ad787bdd46c21fbded3f445df7336 \
+          sha256 b08d174bdaf16baa2d6b614115f3b568587c9e635392899a8d6cc9cf1828a98c \
+          size   133309
```

Normally we would only list rmd160, sha256 and size, and indentations should be at multiples of 4 spaces and should match the rest of the portfile.
```suggestion
checksums               rmd160  1b96ed6a8e0ad787bdd46c21fbded3f445df7336 \
                        sha256  b08d174bdaf16baa2d6b614115f3b568587c9e635392899a8d6cc9cf1828a98c \
                        size    133309
```

---

OK